certbot/CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ More details about these changes can be found on our GitHub repo.
 
 * Added the ability to remove email and phone contact information from an account 
   using `update_account --register-unsafely-without-email`
+* CLI flag `--key-type` has been added to specify 'rsa' or 'ecdsa' (default 'rsa').
+  Only accepts a single value at this time.
+* CLI flag `--elliptic-curve` has been added which takes an NIST/SECG elliptic curve,
+  (see `openssl ecparam -list-curves` for a complete list)
```

Comment:
In the spirit of the discussion above, we should make clear here that only 3 curves are supported.

Reply:
Yes. I have an unpushed change that will list them :)